### PR TITLE
Get ready for 2020

### PR DESCRIPTION
**Pull request by robinpokorny**
2020 is coming tomorrow!

See [String::matchAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll)

### CODE REVIEW

File: src/getAllCSSProperties.ts [hunk 1]
```diff
@@ -9,13 +9,10 @@ export const getAllCSSProperties = () => {
...
-      while ((match = searchRegexp.exec(text))) {
-        accumulator.add(match[2]);
-      }
-      return Array.from(accumulator)
-        .sort()
-        .slice(2);
+      // ES2020 and matchAll FTW!
+      const results = Array.from(text.matchAll(searchRegexp), m => m[2]);
+      // let's use a Set to avoid duplicates HOMIE
+      const uniqeResults = Array.from(new Set(results));
```
Comment (manuelbieh) [inline src/getAllCSSProperties.ts]: fix typo: `uniqe` => `unique` 😉

File: src/getAllCSSProperties.ts [hunk 2]
```diff
@@ -9,13 +9,10 @@ export const getAllCSSProperties = () => {
...
+      return uniqeResults.sort().slice(2);
```
Comment (manuelbieh) [inline src/getAllCSSProperties.ts]: `uniqe` => `unique`

File: src/getAllCSSProperties.ts (hunk 1 above)
Comment (robinpokorny) [reply src/getAllCSSProperties.ts]: OMG!

File: tsconfig.json [hunk 3]
```diff
@@ -2,9 +2,9 @@
   "compilerOptions": {
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```
Comment (manuelbieh) [inline tsconfig.json:5]: if the compile target is ES2020, does it still run in node versions that don't support `.matchAll()`?

Comment (robinpokorny) [reply tsconfig.json:5]: No, this is not just a syntactic sugar and it cannot be transpiled. That's why engine is defined to be 12 or more.
For a production lib, I wouldn't use it. But for this one? 🤷‍♂️

**Review by styfle — APPROVED** [review]

**Review by TejasQ — APPROVED** [review]
I mean if @styfle approves...

**Pull request merged**